### PR TITLE
decode file list for setup.py on python2

### PR DESCRIPTION
**Pull request by gtmanfred**
### What does this PR do?
Decode file list for setup.py

### What issues does this PR fix or reference?
Unable to build `python setup.py sdist` with python2

### Tests written?

No

### Commits signed with GPG?

Yes

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -522,7 +522,11 @@ def make_release_tree(self, base_dir, files):
             self.run_command('write_salt_ssh_packaging_file')
             self.filelist.files.append(os.path.basename(PACKAGED_FOR_SALT_SSH_FILE))
 
-        sdist.make_release_tree(self, base_dir, files)
+        def decode_files(files):
+            PY3 = sys.version_info[0] == 3
```
Comment (s0undt3ch) [inline setup.py]: Better reverse the logic, if py2, because when py4 comes out this will break. Makes sense?

Comment (gtmanfred) [reply setup.py]: that is a solid point

Comment (gtmanfred) [reply setup.py]: actually, it turnes out that IS_PY3 is already a thing in setup.py, so I am going to use that instead.

It just checks if the version is > (3,) so it should still work with py4

**Review by s0undt3ch — APPROVED** [review]
:+1:

**Pull request merged**